package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-api-definition",
-  "version": "3.0.0",
+  "version": "4.0.0-alpha.1",
```

Comment:
I think there is no need to bump up the major version, considering that 
 - `loopback-api-definition` depends only on `specgen` part of `loopback-swagger`
 - the breaking changes in `loopback-swagger@4` were related only to `codegen`